osdep/io.h

```diff
@@ -59,6 +59,15 @@ char *mp_to_utf8(void *talloc_ctx, const wchar_t *s);
 #include <io.h>
 #endif
 
+#ifdef _WIN32
+#include <windows.h>
+#define dlopen(filename, flags) LoadLibrary(TEXT(filename))
```

Comment:
Would fail if the filename argument is not a string literal, so you'd have to add UTF-8 conversion. Also it shouldn't include windows.h - this should be hidden in io.c.

Also not sure if this should be under _WIN32 or __MINGW32__. I think Cygwin does provide dlsym, so this should be under __MINGW32__?

If you think this is unexpectedly too much work, feel free to drop the patch.